readme.mkd

```diff
@@ -190,6 +191,37 @@ If you need secure communication, e.g. are syncing across the open Internet with
 
 This will have the mirror client send traffic to the `localhost:49172` port, which SSH will securely tunnel to your remote host.
 
+Running with Docker
+===================
+
+A docker image is available at [quay.io/stephenh/mirror](https://quay.io/stephenh/mirror).
+
+Since the container will have its own filesystem separate from the host's filesystem,
+usually you'll want to mount some directory *into* the container to make it available for
+synchronisation.
+To mount the current working directory as `/data` into the container, pass `-v $(pwd):/data` to docker.
+
+The image accepts two environment variables to control the owner of the files that *mirror* will create:
+- `$U`: The UID of the user for creating files.
+- `$G`: The GID of the group for creating files.
+
+So for creating files with the current user/group, pass `-eU=$(id -u) -eG=$(id -g)` to docker, when starting the container.
+
+To start a **_mirror_ server** available on port 49172, with the current working directory mounted at `/data` run:
```

Comment:
Looks great. What do you think about making some scripts, e.g.:

```
./docker-server.sh <path>
./docker-client.sh <path>
```

That are just ~1-liners (plus maybe default arg handling) for these two longer `docker run ...` commands?